app/server.go

```diff
@@ -265,6 +303,8 @@ func makeHeaderMatcher(c *Config) func(key string) (string, bool) {
 		switch strings.ToLower(key) {
 		case strings.ToLower(c.AuthenticatedUserHeaderKey):
 			return key, true
+		case "x-trace-id": // TODO: make key configurable
```

Comment:
Is it intentionally will be done later or this is just missing in this PR?

Reply:
fixing it